### PR TITLE
Feature/oxford brca

### PR DESCRIPTION
**Pull request by lauramccluskey1**
Functionality to add test status of 10 to variants with variantpathclass of 1 or 2. Unittests have been updated and run ok for the oxford importer. Couple of integration issues when running all unittests- these have been flagged. QA has been completed.

### CODE REVIEW

File: lib/import/brca/providers/oxford/oxford_handler.rb [hunk 1]
```diff
@@ -180,19 +189,19 @@ def targeted_scope_from_nullscope(genotype, record)
...
               genotype.add_gene_location(exemptions.gsub(/[\[\]+=]+/, ''))
-              genotype.add_status(2)
+              non_pathogenic_variant?(genotype, variantpathclass)
             elsif exemptions.scan(/(?<delinsdup>del|ins|dup)/i).size.positive?
               genotype.add_variant_type($LAST_MATCH_INFO[:delinsdup])
               if exemptions.scan(/(?<exno>[0-9]+-[0-9]+)/i).size.positive?
                 genotype.add_exon_location($LAST_MATCH_INFO[:exno])
               end
-              genotype.add_status(2)
+              non_pathogenic_variant?(genotype, variantpathclass)
```
Comment (shilpigoeldev) [inline lib/import/brca/providers/oxford/oxford_handler.rb:204]: method call for non_pathogenic_variant?(genotype, variantpathclass) can be taken out of if-elsif if in both cases its being called.

**Review by shilpigoeldev — CHANGES_REQUESTED** [review]

File: lib/import/brca/providers/oxford/oxford_handler.rb [hunk 2]
```diff
@@ -12,9 +12,9 @@ def process_fields(record)
                                             record.raw_fields,
                                             PASS_THROUGH_FIELDS)
             assign_test_scope(genotype, record)
-            extract_variantpathclass(genotype, record)
+            variantpathclass = extract_variantpathclass(genotype, record)
```
[the remark this answers is not in the document]
Comment (lauramccluskey1) [reply lib/import/brca/providers/oxford/oxford_handler.rb:15]: This has been updated - see commit f1aa39b

File: lib/import/brca/providers/oxford/oxford_handler.rb (hunk 1 above)
Comment (lauramccluskey1) [reply lib/import/brca/providers/oxford/oxford_handler.rb]: This has been updated - see commit f1aa39b

**Review by shilpigoeldev — CHANGES_REQUESTED** [review]

**Review by shilpigoeldev — COMMENTED** [review]
Thanks , LGTM

**Pull request merged**